pages/builds/environment_variables.md.erb

```diff
@@ -4,51 +4,228 @@ When the agent invokes your build scripts it passes in a set of standard Buildki
+  </tr>
+  <tr>
+    <th><code>BUILDKITE_COMMAND</code></th>
+    <td>The command that will be run for the job. <p class="Docs__api-param-eg"><em>Example:</em> <code>"script/buildkite/specs"</code></p></td>
+  </tr>
+  <tr>
+    <th><code>BUILDKITE_COMMIT</code></th>
+    <td>The commit of the build. <p class="Docs__api-param-eg"><em>Example:</em> <code>"83a20ec058e2fb00e7fa4558c4c6e81e2dcf253d"</code></p></td>
+  </tr>
+  <tr>
+    <th><code>BUILDKITE_JOB_ID</code></th>
+    <td>Internal UUID Buildkite uses for this job. <p class="Docs__api-param-eg"><em>Example:</em> <code>"e44f9784-e20e-4b93-a21d-f41fd5869db9"</code></p></td>
+  </tr>
+  <tr>
+    <th><code>BUILDKITE_LABEL</code></th>
+    <td>The label/name of the current job. <p class="Docs__api-param-eg"><em>Example:</em> <code>"\:hammer\: Specs"</code></p></td>
+  </tr>
+  <tr>
+    <th><code>BUILDKITE_MESSAGE</code></th>
+    <td>The message associated with the build, usually the commit message. <p class="Docs__api-param-eg"><em>Example:</em> <code>"Added a great new feature"</code></p></td>
+  </tr>
+  <tr>
+    <th><code>BUILDKITE_NO_LOCAL_HOOKS</code></th>
+    <td>Whether local hooks in the repository are allowed. <p class="Docs__api-param-eg"><em>Example:</em> <code>"true"</code> or <code>"false"</code></p></td>
+  </tr>
+  <tr>
+    <th><code>BUILDKITE_ORGANIZATION_SLUG</code></th>
+    <td>The organization name on Buildkite as used in URLs. <p class="Docs__api-param-eg"><em>Example:</em> <code>"acme-inc"</code></p></td>
+  </tr>
+  <tr>
+    <th><code>BUILDKITE_PIPELINE_DEFAULT_BRANCH</code></th>
+    <td>The default branch for this pipeline. <p class="Docs__api-param-eg"><em>Example:</em> <code>"master"</code></p></td>
+  </tr>
+  <tr>
+    <th><code>BUILDKITE_PIPELINE_PROVIDER</code></th>
+    <td>The ID of the source code provider for the pipeline’s repository. <p class="Docs__api-param-eg"><em>Example:</em> <code>"github"</code></p></td>
+  </tr>
+  <tr>
+    <th><code>BUILDKITE_PIPELINE_SLUG</code></th>
+    <td>The pipeline slug on Buildkite as used in URLs. <p class="Docs__api-param-eg"><em>Example:</em> <code>"my-project"</code></p></td>
+  </tr>
+  <tr>
+    <th><code>BUILDKITE_PULL_REQUEST</code></th>
+    <td>The number of the pull request if this branch is a pull request. <p class="Docs__api-param-eg"><em>Example:</em> <code>"123"</code> for pull request #123, or <code>"false"</code> if not a pull request.</p></td>
+  </tr>
+  <tr>
+    <th><code>BUILDKITE_PULL_REQUEST_BASE_BRANCH</code></th>
+    <td>The base branch that the pull request is targetting. <p class="Docs__api-param-eg"><em>Example:</em> <code>"master"</code>, or <code>""</code> if not a pull request.</p></td>
+  </tr>
+  <tr>
+    <th><code>BUILDKITE_PULL_REQUEST_REPO</code></th>
+    <td>The repository URL of the third-party pull request. <p class="Docs__api-param-eg"><em>Example:</em> <code>"git://github.com/acme-inc/my-project.git"</code>, or <code>""</code> if not a pull request.</p></td>
```

Comment:
> The repository URL of the third-party pull request

I'm pretty sure this is present for all pull requests, not just third-party ones